src/Summoner/Template.hs

```diff
@@ -117,8 +147,9 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
           exposed-modules:     $libModuleName
                                $preludeMod
           ghc-options:         -Wall
+                               $ghcOptions
           build-depends:       $base
-                             $customPreludePack
+                               $customPreludePack
```

Comment:
You don't need to add extra 2 spaces here. `customPreludePack` variable contains `", "` at the beginning